static/webpack.config.js

```diff
@@ -101,9 +101,9 @@ const config = {
       __dirname + "/js/apps/event/main.ts",
       __dirname + "/css/event.scss",
     ],
-    disaster_dashboard: [
-      __dirname + "/js/apps/disaster_dashboard/main.ts",
-      __dirname + "/css/disaster_dashboard.scss",
+    subject_page: [
```

Comment:
wondering if we should keep disaster dashboard and sustainability explorer files separate on the client side since the two routes are still different from the server and since they're technically independent apps?

Reply:
I think since it sounds like there will still be some experimentation with what we want on each of these pages, it might be better to keep the two separate to allow for possible diversion. And once things are stable and we see that there isn't much diversion, we can refactor?